compression/compression-including-empty-chunk.any.js

```diff
@@ -0,0 +1,57 @@
+// META: global=worker
+// META: script=pako/pako_inflate.min.js
+// META: timeout=long
+
+'use strict';
+
+// This test asserts that compressing '' doesn't affect the compressed data.
+// ex) compressing ['Hello', '', 'Hello'] results in 'HelloHello'
+
+async function compressIncludingEmptyChunk(chunkList, format) {
```

Comment:
The function doesn't actually require there to be an empty chunk. So you could give it a simpler name like "compressChunkList".

Reply:
Okay! Thanks for your advice!